### ISSUE TITLE
请问，如果想输出每个标签对应的score，应该怎么修改呢

### ISSUE THREAD

**Issue by chentutu** [xmxoxo/BERT-train2deploy#20]

**Comment by lc222** [comment]
我现在也遇到这个问题，请问解决了吗

**Comment by LeoWood** [comment]
修改bert_base安装目录中的..\Lib\site-packages\bert_base\server\__init__.py文件：
line 465：
`return EstimatorSpec(mode=mode, predictions={
    'client_id': features['client_id'],
    'encodes': tf.argmax(pred_probs[0], axis=-1),
    'score': tf.reduce_max(pred_probs[0], axis=-1)
})`
修改为：
`return EstimatorSpec(mode=mode, predictions={
    'client_id': features['client_id'],
    'encodes': pred_probs[0],
    'score': pred_probs[0]
})`
line 515:
`pred_label_result = [self.id2label.get(x, -1) for x in r['encodes']]`
修改为：
`pred_label_result = r['encodes'].tolist()`

**Comment by lc222** [comment]
嗯嗯，感谢。我也大概是这么修改的~~只不过515改成了：
pred_label_result = [self.id2label.get(y, -1) for x in r['encodes'] for y in range(class_num)]

不过还有一个地方，如果不注释掉后面会报错。350行，虽然注释掉可能会有其他并发的问题，但是我先这么改了
finished = [(k, v) for k, v in pending_result.items()] #if pending_checksum[k] == job_checksum[k]]
